railties/test/application/middleware/flash_test.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require "isolation/abstract_unit"
+require "rack/test"
+
+module ApplicationTests
+  class FlashTest < ActiveSupport::TestCase
+    include ActiveSupport::Testing::Isolation
```

Comment:
Hey! So, if I am understanding this file correctly, this will create a fork prior to loading `action_dispatch/middleware/flash.rb` so that the Request class is clean for the tests in this class? That is pretty snazzy.

Reply:
> so that the Request class is clean for the tests in this class

That's correct :)